tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/Emitter.scala

```diff
@@ -462,4 +462,56 @@ object Emitter {
       value
     }
   }
+
+  private def makeSymbolRequirements(semantics: Semantics, esLevel: ESLevel): SymbolRequirement = {
+    import semantics._
+    import CheckedBehavior._
+
+    val factory = SymbolRequirement.factory("corejslib")
```

Comment:
Hmmm... how about `"emitter"`? Not everything in there is really referenced in "scalajsenv.js", right?